src/components/application_manager/include/application_manager/application.h

```diff
@@ -934,6 +935,52 @@ class Application : public virtual InitialApplicationData,
    */
   virtual const std::list<AppExtensionPtr>& Extensions() const = 0;
 
+  /**
+   * @brief Get map of pending button subscription requests correlation ids
+   * to button names
+   * @return pending button subscriptions map
+   */
+  const virtual std::map<int32_t, hmi_apis::Common_ButtonName::eType>&
```

Comment:
@mked-luxoft I think we can use typedef here like a `MapButtons`

Reply:
@ZhdanovP fixed in 704a3e5e302aba1557830edf5e6afedb18c317e3